core/domain/skill_services.py

```diff
@@ -230,6 +232,167 @@ def get_all_skill_summaries():
     return skill_summaries
 
 
+def get_filtered_skill_dicts(status, classroom_name, keywords, sort_by):
+    """Returns all the skill summary dicts after filtering.
+
+    Args:
+        status: string. The status of the skill.
```

Comment:
Do we use "string" elsewhere, in the Python code? I thought the convention was str or unicode.

Reply:
Done.